rockfield/sass/_config-child-theme-deep.scss

```diff
@@ -266,8 +266,8 @@ $config-list: (
 	// Fonts
 	"font": (
 		"family": (
-			"fallback": map-deep-get($config-global, "font", "family", "primary", "fallback"),
-			"css-var": map-deep-get($config-global, "font", "family", "primary", "css-var"),
+			"fallback": map-deep-get($config-global, "font", "family", "secondary", "fallback"),
+			"css-var": map-deep-get($config-global, "font", "family", "secondary", "css-var"),
```

Comment:
Are these lines actually used?

Reply:
Umm, yes. They compile to:


```
ul,
ol {
	font-family: Lora, Baskerville, Georgia, Times, serif;
	font-family: var(--font-base, Lora, Baskerville, Georgia, Times, serif);
}
```

Reply:
There's another issue regarding this: #2282 - questioning whether this should be set at all.

Reply:
Yeah that's what I'm wondering. It feel unnecessary. Can we just remove it?

Reply:
We can remove it, but it seems that Rockfield/Varia will, regardless, compile and output code related to this:
```
ul,
ol {
	font-family: serif;
	font-family: var(--font-base, serif);
	margin: 0;
	padding-left: 32px;
}

```

Reply:
Do you know where that first `font-family: serif;` comes from?

Reply:
Both font-family's come from https://github.com/Automattic/themes/blob/3fa9d68c06fb4d174aebe58f58952061839efbb2/varia/sass/blocks/list/_style.scss#L3

Reply:
Looks like its there for IE11 support. Does this solution solve the original issue?

Reply:
Both options seem to be working fine: changing the family from `primary` to `secondary` or removing the code altogether.

Reply:
In that case let's remove the code. Less code = less bugs!